packages/mongodb-rag-core/src/index.ts

```diff
@@ -25,3 +25,4 @@ export * from "./References";
 export * from "./VectorStore";
 export * from "./arrayFilters";
 export * from "./assertEnvVars";
+export * from "./mongoDbMetadata";
```

Comment:
```suggestion
```

pls do this as a named export. in the PR #678, i added much of the metadata stuff. 

it's exported as `mongodb-rag-core/mongoDbMetadata`. 

pls update the imports throught this PR accordingly.